src/main/kotlin/frc/robot/subsystems/drive/Drive.java

```diff
@@ -343,23 +420,53 @@ public double getFFCharacterizationVelocity() {
         return output;
     }
 
-    /** Returns the current odometry pose. */
+    /**
+     * Returns the current odometry pose.
+     */
     @AutoLogOutput(key = "Odometry/Robot")
     public Pose2d getPose() {
         return poseEstimator.getEstimatedPosition();
     }
 
-    /** Returns the current odometry rotation. */
+    /**
+     * Returns the current odometry rotation.
+     */
     public Rotation2d getRotation() {
         return getPose().getRotation();
     }
 
-    /** Resets the current odometry pose. */
+    public Command updateDesiredHeading(DoubleSupplier omegaAxis) {
+        return Commands.run(
+                () -> {
+                    double desiredDeltaOmega =
+                            MathUtil.applyDeadband(omegaAxis.getAsDouble(), 0.15)
+                                    * TunerConstants.kMaxOmegaVelocity.in(RadiansPerSecond)
+                                    * ConstantsKt.LOOP_TIME;
```

Comment:
It'll be more accurate to calculate the difference since the last loop.

Reply:
34afd579ac1caf574c360e7c070b8615e9c82aa1